src/odemis/driver/andorcam2.py

```diff
@@ -538,12 +538,15 @@ def __init__(self, name, role, device=None, emgains=None, shutter_times=None,
         # Strong cooling for low (image) noise
         if self.hasSetFunction(AndorCapabilities.SETFUNCTION_TEMPERATURE):
             if self.hasGetFunction(AndorCapabilities.GETFUNCTION_TEMPERATURERANGE):
-                ranges = self.GetTemperatureRange()
+                trange = self.GetTemperatureRange()
             else:
-                ranges = (-275, 100)
-            self.targetTemperature = model.FloatContinuous(ranges[0], ranges, unit=u"°C",
+                trange = (-275, 25)
+            self._hw_temp_range = trange
+            # Always support 25°C, to disable the cooling
+            trange = (trange[0], max(trange[1], 25))
+            self.targetTemperature = model.FloatContinuous(trange[0], trange, unit=u"°C",
                                                            setter=self._setTargetTemperature)
-            self._setTargetTemperature(ranges[0], force=True)
+            self._setTargetTemperature(trange[0], force=True)
```

Comment:
Is this still necessary? I thought the value is set when you create the FloatContinuous VA

Reply:
It is. Actually, the initial value of a VA *is not* passed to the setter. That's on purpose, because in many cases, the VA is set with the current value (of the hardware), so it doesn't make sense to call the setter immediately.